client/iothub/client.go

```diff
@@ -250,30 +250,36 @@ func (c *client) DeleteDevice(ctx context.Context, cs *model.ConnectionString, i
 }
 
 func (c *client) GetDeviceTwins(
-	ctx context.Context, cs *model.ConnectionString,
-) (Cursor, error) {
-	const (
-		SQLQuery              = `{"query":"SELECT * FROM devices"}`
-		pageSize, pageSizeStr = 100, "100"
+	ctx context.Context, cs *model.ConnectionString, deviceIDs []string,
```

Comment:
is there a limit for number of devices iothub can return?

Reply:
I think it is `1000`, but the documentation doesn't really say, but this is the maximum number of devices returned by the `GET /devices`. I will test with 500 just to be sure.

Reply:
I can confirm, the API indeed allows more than 500 items:
```
$ curl https://[...].azure-devices.net/devices/query?api-version=2021-04-12 [...] -d '{"query": "SELECT * FROM devices"}' -H 'x-ms-max-item-count: 501' | jq '. | length'
501
```